### ISSUE TITLE
Update font media types

### ISSUE THREAD

**Issue by alrra** [jshttp/mime-db#77]
The [`font` top-level media type](https://tools.ietf.org/html/rfc8081) has been registered, so the new media types for font files are as follows:

| Font format | Media type |
|:---|:---|
| `collection` | `font/collection` |
| `otf` | `font/otf` |
| `sfnt` | `font/sfnt` |
| `ttf` | `font/ttf` |
| `woff` |	`font/woff` | 	
| `woff2` | `font/woff2` |

See also: https://www.iana.org/assignments/media-types/media-types.xhtml#font

---

@dougwilson I did run the `fetch-iana` script but it doesn't seem to get these updates. Let me know if you are open to updating `custom.json`.

**Comment by sholladay** [comment]
This would be a nice change as it is simpler and more intuitive. I am curious about browser support.

**Comment by alrra** [comment]
>  I am curious about browser support.

I [responded](https://twitter.com/alrra/status/744966378160070656) to a now deleted tweet about that. 

TL;DR: browsers ignore font media types.

**Comment by dougwilson** [comment]
> TL;DR: browsers ignore font media types.

Is that all browsers or only current browsers? I'd be interesting to know about current Firefox, current Firefox ESR, current Chrome, current Safari, current Edge, current IE, IE10 (probably IE9, but no lower since I think SSL is broken now on those anyway). Anything else @sholladay you would like to know about?

**Comment by sholladay** [comment]
I only personally care about latest of: Chrome, Firefox, Safari, Edge

FYI Microsoft dropped support for IE < 11 early last year and they are no longer even remotely secure, as they stopped receiving basic security patches. IE11 has approximately 13% market share and falling quickly. It is a new day!

I know the situation with fonts on mobile has traditionally been very different than desktop (e.g. for a while only SVG was supported). It wouldn't surprise me if they avoided sniffing to save battery. That is probably a bigger concern. But following the standard is a 👍 from me.

**Comment by alrra** [comment]
> Is that all browsers or only current browsers? 

@dougwilson The only problem was with old Blink/WebKit based browsers, which used to [display a warning(https://github.com/h5bp/server-configs-apache/issues/32#issuecomment-47935393), but that is [no longer the case](https://bugs.chromium.org/p/chromium/issues/detail?id=460669).

**Comment by dougwilson** [comment]
I wonder when Apache / NGINX will update their mappings to use these new types. I'm going to see if there are bug reports in those projects. That's where our font types are coming from today, if you were wondering :)

**Comment by alrra** [comment]
>  I'm going to see if there are bug reports in those projects. 

@dougwilson Did you manage to look into that? Thanks!

Also, in the meantime, 

> Let me know if you are open to updating custom.json.

**Comment by dougwilson** [comment]
Hi @alrra I didn't check yet. I can take a look now. I would prefer this remain simply an aggregation system, as in it just passes through the other sources, so unless there is some specific reason (like types that came into existence before IANA and similar, as one example) I would rather not add things to custom.json . Our font data comes from the Apache / NGINX projects currently, so updating those projects will mean we'll pull down the new mappings.

**Comment by dougwilson** [comment]
A quick search didn't find anything in the Apache HTTPD bug tracker, but found https://trac.nginx.org/nginx/ticket/1243 for NGINX.

**Comment by dougwilson** [comment]
@alrra you're welcome to make a pull request to get the types added, though that will just list the existing extensions twice until the upstream removes them.

**Comment by alrra** [comment]
>  you're welcome to make a pull request to get the types added,

[Done](https://github.com/jshttp/mime-db/pull/90).

> though that will just list the existing extensions twice until the upstream removes them.

:(

**Comment by dougwilson** [comment]
Alright, Apache changed their types now, which makes landing this fit very nicely now :O

**Comment by alrra** [comment]
> Alright, Apache changed their types now, which makes landing this fit very nicely now :O

@dougwilson I've updated #90.
